docs/03-03-examples-alpha.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Kyma CLI alpha command usage examples
+type: Details
+---
+
+The following examples show how to use the alpha commands to provision a cluster, install and upgrade Kyma, and handle errors.
+
+## Provision a cluster
+To provision a Kubernetes cluster, run:
```

Comment:
I would make it more specific, a k3s cluster or a local cluster

Reply:
```suggestion
To provision a k3s cluster, run:
```